src/test/java/com/answerdigital/answerking/service/OrderServiceTest.java

```diff
@@ -56,16 +56,14 @@ class OrderServiceTest {
 
     private static final Long NONEXISTENT_ORDER_ID = 10L;
 
-    private static final Long ORDER_ID = 1L;
-
     @Test
     void testAddOrderWithNoProductsValidOrderRequestIsSuccessful() {
         // Given
         final Order order = orderTestBuilder
             .withDefaultValues()
             .build();
         final OrderRequest orderRequest = orderRequestTestBuilder
-            .withDefaultValues()
+            .withLineItemRequests(new ArrayList<>())
```

Comment:
I'd recommend we create the OrderRequest with both of these lines, as initialising it this way looks to me as though it'll create an OrderRequest with everything else being null.